rust/src/cliwrap.rs

```diff
@@ -138,11 +137,17 @@ fn write_wrappers(rootfs_dfd: &Dir) -> Result<()> {
     rootfs_dfd.ensure_dir_with(destdir.parent().unwrap(), &dirbuilder)?;
     rootfs_dfd.ensure_dir_with(destdir, &dirbuilder)?;
 
+    // Binaries we will wrap, or create if they don't exist.
+    let mut must_wrap_binaries: &[&str] = &["usr/bin/yum", "usr/bin/dnf"];
+    if ostree_ext::container_utils::is_ostree_container()? {
```

Comment:
So one problem here is that this won't work if we changed the default build to enable cliwrap.  By "default build" I mean `rpm-ostree compose tree` which is generating the "base image", and if we set `cliwrap: true` in the manifest.

Hmm.  I think I'd say we should just add `kernel-install` to the unconditional list.
I can't see any problem with that. 

(But...our `kernel_install.rs` code could *itself* detect whether or not it's in an ostree container)